src/workloads/query/TimeseriesBlockProcessing.yml

```diff
@@ -0,0 +1,356 @@
+
+  - Name: AndMatchOnNestedField
+    Type: CrudActor
+    Database: *database
+    Threads: *threads
+    Phases:
+      OnlyActiveInPhases:
+        Active: [6]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          $and:
+                            [
+                              { "obj.measurement3": { $gt: 100 } },
+                              { "obj.measurement3": { $lt: 300 } },
+                            ],
+                        },
+                    },
+                    { $count: "count" },
+                  ]
+
+  - Name: OrMatchOnNestedField
+    Type: CrudActor
+    Database: *database
+    Threads: *threads
+    Phases:
+      OnlyActiveInPhases:
+        Active: [7]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          $or:
+                            [
+                              { "obj.measurement3": { $lt: 100 } },
+                              { "obj.measurement3": { $gt: 900 } },
+                            ],
+                        },
+                    },
+                    { $count: "count" },
+                  ]
+
+  - Name: MatchOnNestedFieldAndTime
+    Type: CrudActor
+    Database: *database
+    Threads: *threads
+    Phases:
+      OnlyActiveInPhases:
+        Active: [8]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          $and:
+                            [
+                              {
+                                "time":
+                                  { $lt: { ^Date: "2022-01-02T00:00:00" } },
+                              },
+                              { "obj.measurement2": { $gte: 25 } },
+                            ],
+                        },
+                    },
+                    { $count: "count" },
+                  ]
+
+  # TODO: The $or $eq on obj.measurement1 gets converted to a $in so this query can't run in
```

Comment:
I'd throw a reference to SERVER-83057 here. After that ticket goes in we can uncomment it, just so we're getting results. Even if it's not using BP it's useful to see where SBE is regressing